engine/src/main/resources/assets/ui/enterTextPopup.ui

```diff
@@ -18,7 +18,7 @@
                     "contents": [
                         {
                             "type": "UILabel",
-                            "text": "Enter name",
+                            "text":  "${engine:menu#player-name}" + ": ",
```

Comment:
Be careful in JSON - you cannot necessarily use the same string building as in Java :-)